src/modules/feature/folder/Folder.vue

```diff
@@ -70,19 +78,29 @@ const {
 
 const { createPageInformation } = useSharedBoardPageInformation();
 
-const { fetchFiles, upload, getFileRecordsByParentId, deleteFiles } =
+const { fetchFiles, upload, getFileRecordsByParentId, deleteFiles, rename } =
 	useFileStorageApi();
 
+const authModule: AuthModule = injectStrict(AUTH_MODULE_KEY);
+const userRoles = ref(authModule.getUserRoles);
+const isStudent = computed(() => {
+	return userRoles.value.includes("student");
+});
+
 const folderId = toRef(props, "folderId");
 const fileRecords = computed(() => getFileRecordsByParentId(folderId.value));
 const fileInput = ref<HTMLInputElement | null>(null);
 
-const fabAction = {
-	icon: mdiPlus,
-	title: t("pages.folder.fab.title"),
-	ariaLabel: t("pages.folder.fab.title"),
-	dataTestId: "fab-add-files",
-};
+const fabAction = computed(() => {
+	if (isStudent.value) return undefined;
```

Comment:
```suggestion
	if (isStudent.value) return;
```
Ich kenn es so, dass man das `undefined` nicht explizit hinschreibt, aber ist Geschmackssache.